granule_metadata_extractor/processing/process_ualbmrr2impacts.py

```diff
@@ -18,7 +18,10 @@ class ExtractUalbmrr2impactsMetadata(ExtractNetCDFMetadata):
     def __init__(self, file_path):
         # super().__init__(file_path)
         self.file_path = file_path
-        self.format = 'netCDF-3'

```

Comment:
For the future: Python's ternary operator isn't the prettiest, but this would be a good use case for it. Essentially you can write a one line if/else condition like the following.
```
self.format = 'netCDF-3' if '_mrr2_2020' in file_path else 'netCDF-4'
```